frontend/services/tools/submitPrompt.js

```diff
@@ -1,29 +1,59 @@
 import axios from 'axios';
 
-const submitPrompt = async (payload, files) => {
+import createToolSession from './createToolSession';
+
+import store from '@/redux/store';
+import fetchToolsHistory from '@/redux/thunks/toolsHistory';
+
+const submitPrompt = async (payload, files, dispatch) => {
   try {
     const formData = new FormData();
-
-    // Append payload to the form data
     formData.append('data', JSON.stringify(payload));
 
-    // Append files to the form data
-    if (!!files && files?.length > 0) {
+    if (files && files.length > 0) {
       files.forEach((file, index) => {
         formData.append(`file${index}`, file);
       });
     }
 
-    const response = await axios.post('/api/tool/', formData, {
+    const url = process.env.NEXT_PUBLIC_KAI_ENDPOINT;
+
+    const response = await axios.post(url, formData, {
       headers: {
         'Content-Type': 'multipart/form-data',
       },
     });
 
+    const toolSessionPayload = {
+      user: payload.user,
+      tool_data: { ...payload.tool_data },
+      type: payload.type,
+      messages: response.data,
+      sessionId: payload.sessionId,
+    };
```

Comment:
Why are we doing this here ?

Reply:
We got response data (output data) in line 21 stored in response. And I stored the output as messages(I will change it to outputs), and send to firestore using createtoolsession function.